test/e2e/storage/testsuites/provisioning.go

```diff
@@ -828,6 +851,8 @@ func prepareSnapshotDataSourceForProvisioning(
 	}
 
 	cleanupFunc := func() {
+		clearComputedStorageClass()
```

Comment:
There is not supposed to be a dependency on StorageClass for volume deletion, but unfortunately we may have skipped through the cracks in some corner cases. So I think this needs to be called at the end. ditto everywhere

Reply:
Ack, I'll move this line to the end